pkg/admin_cluster_services.go

```diff
@@ -317,7 +317,7 @@ func getQoveryClient() (*qovery.APIClient, error) {
 
 func (service AdminClusterBatchDeployServiceImpl) Deploy(clusters []ClusterDetails) (*ClusterBatchDeployResult, error) {
 	if !service.DryRunDisabled {
-		utils.Println("dry-run-disabled is false: following information is purely indicative, no cluster will be deployed at all")
+		utils.Println("dry-run-disabled is false: it is going to trigger deployment of cluster in dry-run mode")
```

Comment:
_wording_: `dry-run-disabled is false: trigger cluster deployment dry-run mode (no changes will be made)`